launch_ros/launch_ros/actions/node.py

```diff
@@ -448,8 +475,10 @@ def execute(self, context: LaunchContext) -> Optional[List[Action]]:
             ros_specific_arguments['name'] = '__node:={}'.format(self.__expanded_node_name)
         if self.__expanded_node_namespace != '':
             ros_specific_arguments['ns'] = '__ns:={}'.format(self.__expanded_node_namespace)
+        if context.launch_configurations.get('__secure', None):
+            self._setup_security(context, ros_specific_arguments)
         context.extend_locals({'ros_specific_arguments': ros_specific_arguments})
-        ret = super().execute(context)
+        super().execute(context)
```

Comment:
Looks like this was meant to return something, can you clarify what you're changing, here?

Reply:
Ah, this was changed for a moment and then back, it always returns none but I forgot to undo this.